cellSAM/cellsam_pipeline.py

```diff
@@ -79,6 +79,70 @@ def cellsam_pipeline(
         iou_threshold=0.5,
         filter_below_min=True, 
 ):
+    """Run the cellsam inference pipeline on `img`.
+
+    Cellsam is capable of segmenting a variety of cells (bacteria,
+    eukaryotic, etc.) spanning all forms of microscopy (brightfield,
+    phase, autofluorescence, electron microscopy) and
+    staining (H&E, PAS, etc.) / multiplexed (codex, mibi, etc.)
+    modalities.
+
+    Parameters
+    ----------
+    img : array_like with shape ``(W, H)`` or ``(W, H, C)``, where C is 1 or 3
+        The image to be segmented. For multiple-channel images, `img` should
+        have the following format:
+
+          - **Stained images (e.g H&E)**: ``(W, H, C)`` where ``C == 3``
+            representing color channels in RGB format.
+          - **Multiplexed images**: ``(W, H, C)`` where ``C == 3`` and the
+            channel ordering is: ``(blank, nuclear, membrane)``. The
+            ``membrane`` channel is optional, in which case a nuclear segmentation
+            is returned.
+    chunks : int
+        TODO: should this be an option?
```

Comment:
AFAICT this option is only used in the dask array layout; however, we're not (currently) using any on-disk features nor per-block parallelism, so is there any benefit in exposing this parameter?

Reply:
Yah, I think it doesn't need to be exposed